Components/Flash/Driver/w25qxx.hpp

```diff
@@ -96,6 +96,7 @@ extern "C"
 	void W25qxx_WritePage(uint8_t *pBuffer, uint32_t Page_Address, uint32_t OffsetInByte, uint32_t NumByteToWrite_up_to_PageSize);
 	void W25qxx_WriteSector(uint8_t *pBuffer, uint32_t Sector_Address, uint32_t OffsetInByte, uint32_t NumByteToWrite_up_to_SectorSize);
 	void W25qxx_WriteBlock(uint8_t *pBuffer, uint32_t Block_Address, uint32_t OffsetInByte, uint32_t NumByteToWrite_up_to_BlockSize);
+	void W25qxx_WritePageScary(uint8_t *pBuffer, uint32_t Page_Address);
```

Comment:
Change IDE to 4-space tabs, get rid of this function declaration unless it's used somewhere